src/Microsoft.DocAsCode.Build.Engine/TemplateProcessors/TemplateProcessor.cs

```diff
@@ -35,6 +35,10 @@ public TemplateProcessor(ResourceFileReader resourceProvider, DocumentBuildConte
         _resourceProvider = resourceProvider;
         _maxParallelism = maxParallelism;
         _templateCollection = new TemplateCollection(resourceProvider, context, maxParallelism);
+        if (_templateCollection.Count == 0)
+        {
+            Logger.LogWarning("No template bundles were found, no template will be applied to the documents. 1) Check your docfx.json 2) or your working directory has the templates subfolder");
```

Comment:
> 2) or your working directory has the templates subfolder

I though current docfx.exe don't use `{cwd}/templates/**` for resource lookup.

https://github.com/dotnet/docfx/blob/c1545fe6f006a41df1551ed53d6b9961bacdc4a8/src/Microsoft.DocAsCode.Build.Engine/TemplateProcessors/TemplateManager.cs#L45-L61

It seems template folder are searched with following orders.
1. Launched EXE directory (`AppContext.BaseDirectory`)
2. `_baseDirectory` that is passed to TemplateProcessor's constructor.

When RunBuild/RunPdf operation is executed. `_baseDirectory` is always set to `docfx.json` existing directory.
So. `cwd` is not used for template resource lookup.